CHANGELOG.md

```diff
@@ -1,5 +1,125 @@
+
+In previous versions, you used the following methods to specify TypeScript compiler options:
+
+* the [--ts-config-path](https://devexpress.github.io/testcafe/documentation/reference/command-line-interface.html#--ts-config-path-path) command line flag
+
+    ```sh
+    testcafe chrome my-tests --ts-config-path path/to/config.json
+    ```
+
+* the [runner.tsConfigPath](https://devexpress.github.io/testcafe/documentation/reference/testcafe-api/runner/tsconfigpath.html) method
+
+    ```js
+    runner.tsConfigPath('path/to/config.json');
+    ```
+
+* the [tsConfigPath](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#tsconfigpath) configuration file property
+
+    ```json
+    {
+        "tsConfigPath": "path/to/config.json"
+    }
+    ```
+
+In v1.10.0, we introduced a new easy-to-use API that allows you to specify the compiler options in the command line, API or TestCafe configuration file, without creating a separate JSON file. The new API is also designed to accept options for more compilers (for instance, Babel) in the future releases.
+
+The API consists of the following members:
+
+* the [--compiler-options](https://devexpress.github.io/testcafe/documentation/reference/command-line-interface.html#--compiler-options-options) command line flag
+
+    ```sh
+    testcafe chrome my-tests --compiler-options typescript.experimentalDecorators=true
+    ```
+
+* the [runner.compilerOptions](https://devexpress.github.io/testcafe/documentation/reference/testcafe-api/runner/compileroptions.html) method
+
+    ```js
+    runner.compilerOptions({
+        typescript: {
+            experimentalDecorators: true
+        }
+    });
+    ```
+
+* the [compilerOptions](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#compileroptions) configuration file property
+
+    ```json
+    {
+        "compilerOptions": {
+            "typescript": {
+                "experimentalDecorators": true
+            }
+        }
+    }
+    ```
+
+If you prefer to keep compiler settings in a configuration file, you can use the new API to specify the path to this file:
+
+```sh
+testcafe chrome my-tests --compiler-options typescript.configPath='path/to/config.json'
+```
+
+In v1.10.0, you can customize TypeScript compiler options only.
+
+For more information, see [TypeScript and CoffeeScript](https://devexpress.github.io/testcafe/documentation/guides/concepts/typescript-and-coffeescript.html).
+
+#### Added a Selector Method to Access Shadow DOM ([PR #5560](https://github.com/DevExpress/testcafe/pull/5560) by [@mostlyfabulous](https://github.com/mostlyfabulous))
+
+This release introduces the [selector.shadowRoot](https://devexpress.github.io/testcafe/documentation/reference/test-api/selector/shadowroot.html) method that allows you to easily access and interact with the shadow DOM elements. This method returns a shadow DOM root hosted in the selector's matched element.
```

Comment:
"that allows you to easily access and interact" I would advise you not to use 'easily' without explaining why it is easily.